roles/openshift_examples/files/examples/x86_64/xpaas-templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -5,19 +5,19 @@
         "annotations": {
             "iconClass": "icon-eap",
             "tags": "eap,javaee,java,jboss",
-            "version": "1.5.0",
-            "openshift.io/display-name": "JBoss EAP CD + AMQ 7 (with https)",
+            "version": "14.0",
```

Comment:
Same here - latest tag is 15, but version is 14?

Reply:
@vrutkovs thanks and sorry for wasting your time by missing that.